app/settings/fiware-glancesync.cfg

```diff
@@ -0,0 +1,59 @@
+# -*- encoding: utf-8 -*-
+#
+# Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-WARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+
+[openstack]
+OPENSTACK_URL: http://cloud.lab.fiware.org:4730/v2.0
+ADM_USER: admin
+ADM_PASS: zMCGfpTLTAMF2L2c
+ADM_TENANT_ID:
+ADM_TENANT_NAME: admin
+USER_DOMAIN_NAME: Default
+AUTH_API_V3: v3
+AUTH_API_V2: v2.0
+
+[glancesync]
+SETTINGS_TYPE: production
+OWNER: Telefonica I+D
+API_INFO_URL: http://docs.glancesync.apiary.io/#
+VERSION: 1.1.0
+STATUS: SUPPORTED
+UPDATED: 2015-12-12T13:16:21Z
+PORT: 8080
+HOST: 0.0.0.0
+
+[logging]
+level: INFO
+# LOGGING_PATH: /var/log/fiware-glancesync
+LOGGING_PATH: .
+
+[http]
+CONTENT_TYPE: Content-Type
+ACCEPT: Accept
+JSON_TYPE: application/json
+X_AUTH_TOKEN_HEADER: X-Auth-Token
+X_SUBJECT_TOKEN_HEADER: X-Subject-Token
+TOKENS_PATH_V2: tokens/
+TOKENS_PATH_V3: auth/tokens/
+SERVER: Werkzeug
+SERVER_HEADER: server
```

Comment:
Please add a short comment before each parameter, so that configuration file is self-documented